### ISSUE TITLE
kroki.io site seems down (2023-03-29)

### ISSUE THREAD

**Issue by gissehel** [yuzutech/kroki#1498]
Similarly to what happens on 2023-03-08, ( issue #1457 ) the site is down, at least for the "dynamic part".

![image](https://user-images.githubusercontent.com/199364/228464709-ca7beb97-075c-4c89-a901-8c9c397290b2.png)

Also the error was seen on both my personal ISP and my professional access.

**Comment by ggrossetie** [comment]
Thanks for your report!

I think there's also a memory leak in PlantUML: https://github.com/plantuml/plantuml/issues/1368
The next version of Kroki will call both UMlet and PlantUML as binaries (thus preventing OutOfMemory errors on the gateway server).

**Comment by ggrossetie** [comment]
Service is up and running again

**Comment by kolovos** [comment]
Kroki.io returns a 502 error at the moment for the sample URL provided on the Kroki website.

https://kroki.io/plantuml/svg/eNpljzEPgjAQhff-iguTDFQlcYMmuru5mwNO0tCWhjY6GP-7LRJTdHvv7r67d26QxuKEGiY0gyML5Y65b7GzEvblIalYbAfs6SK9oqOSvdFkPCi6ecYmaj2aXhFkZ5QmgycD2Ogg-V3SI4_OyTjgR5OzVwqc0NECNEHydtR2NGH3TK2dHjtSP3zViPmQd9W2ERmgg-iv3jGW4MC5-L-wTEJdi1XeRENRiFWOtMfnrclriQ5gJD-Z3x9beAM=

![image](https://user-images.githubusercontent.com/3584136/232058702-fb71735f-7659-4517-a2b8-4ac2acb479c4.png)

**Comment by ggrossetie** [comment]
Should be up and running again

**Comment by kolovos** [comment]
That's great. Thank you very much for the quick reaction and for all your hard work on Kroki!

**Comment by FD-HH** [comment]
Service seems to be down again, I see gateway errors for the example 
https://kroki.io/plantuml/svg/eNpljzEPgjAQhff-iguTDFQlcYMmuru5mwNO0tCWhjY6GP-7LRJTdHvv7r67d26QxuKEGiY0gyML5Y65b7GzEvblIalYbAfs6SK9oqOSvdFkPCi6ecYmaj2aXhFkZ5QmgycD2Ogg-V3SI4_OyTjgR5OzVwqc0NECNEHydtR2NGH3TK2dHjtSP3zViPmQd9W2ERmgg-iv3jGW4MC5-L-wTEJdi1XeRENRiFWOtMfnrclriQ5gJD-Z3x9beAM=

**Comment by ggrossetie** [comment]
Yes, we are experiencing some instabilities on PlantUML and UMlet. Hopefully, the next version will fix this recurring issue.
The service should now be up and running.
